packages/network-of-terms-catalog/catalog/queries/search/rkdartists.rq

```diff
@@ -1,28 +1,68 @@
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX schema: <http://schema.org/>
+PREFIX schema: <https://schema.org/>
 PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 
 CONSTRUCT {
     ?uri a skos:Concept ;
-        skos:prefLabel ?schema_name ;
-        skos:altLabel ?schema_alternateName ;
-        skos:scopeNote ?schema_description ;
-        rdfs:seeAlso ?rdfs_seeAlso .
+        skos:prefLabel ?prefLabel ;
+        skos:altLabel ?altLabel ;
+        skos:scopeNote ?scopeNote ;
+        rdfs:seeAlso ?seeAlso ;
+        skos:label ?label .
 }
 WHERE {
-    ?uri a ?type .
-    VALUES ?type { schema:Person schema:Organization }
-    ?uri ?name ?label .
-    VALUES ?name { schema:name schema:alternateName }
-
     ?uri ?predicate ?label .
     ?label <bif:contains> ?virtuosoQuery .
```

Comment:
I’m not sure that we should search _all_ fields. If you search for ‘Vincent Van Gogh’ you’ll find many not so relevant results that mention Van Gogh in their scopeNote. Should we only include schema:name and schema:alternateName?

Reply:
I was also wondering if we should keep this in. I will check with RKD

Reply:
I limited this for now to `schema:name` and `schema:alternateName` for parity with the old endpoint. If needed, we can revert that later.